src/elements/content-sidebar/activity-feed/comment/Replies.js

```diff
@@ -0,0 +1,160 @@
+// @flow
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import noop from 'lodash/noop';
+
+import BaseComment from './BaseComment';
+import LoadingIndicator from '../../../../components/loading-indicator';
+import messages from './messages';
+import PlainButton from '../../../../components/plain-button';
+
+import type { GetAvatarUrlCallback, GetProfileUrlCallback } from '../../../common/flowTypes';
+import type { Translations } from '../../flowTypes';
+import type { SelectorItems, User } from '../../../../common/types/core';
+import type { BoxCommentPermission, Comment as CommentType, FeedItemStatus } from '../../../../common/types/feed';
+
+import './Replies.scss';
+
+type ReplyProps = {
+    hasReplies?: boolean,
+    isAlwaysExpanded?: boolean,
+    isRepliesLoading?: boolean,
+    onHideReplies?: (lastReply: CommentType) => void,
+    onReplyCreate?: (text: string) => void,
+    onReplyDelete?: ({ id: string, permissions: BoxCommentPermission }) => void,
+    onReplyEdit?: (
+        id: string,
+        text: string,
+        status?: FeedItemStatus,
+        hasMention?: boolean,
+        permissions: BoxCommentPermission,
+        onSuccess: ?Function,
+        onError: ?Function,
+    ) => void,
+    onReplySelect?: (isSelected: boolean) => void,
+    onShowReplies?: () => void,
+    parentID: string,
+    replies?: CommentType[],
+    repliesTotalCount?: number,
+};
+
+type Props = ReplyProps & {
+    currentUser?: User,
+    getAvatarUrl: GetAvatarUrlCallback,
+    getMentionWithQuery?: Function,
+    getUserProfileUrl?: GetProfileUrlCallback,
+    mentionSelectorContacts?: SelectorItems<>,
+    translations?: Translations,
+};
+
+const Replies = ({
+    currentUser,
+    getAvatarUrl,
+    getMentionWithQuery,
+    getUserProfileUrl,
+    hasReplies = false,
+    isAlwaysExpanded = false,
+    isRepliesLoading,
+    mentionSelectorContacts,
+    onHideReplies = noop,
+    onReplyDelete = noop,
+    onReplyEdit = noop,
+    onReplySelect = noop,
+    onShowReplies = noop,
+    parentID,
+    replies = [],
+    repliesTotalCount = 0,
+    translations,
+}: Props) => {
+    const { length: repliesLength } = replies;
+    const repliesToLoadCount = Math.max(repliesTotalCount - repliesLength, 0);
+
+    const onHideRepliesHandler = () => {
+        if (repliesLength) {
+            onHideReplies(replies[repliesLength - 1]);
+        }
+    };
+
+    const renderButton = () => {
+        if (isAlwaysExpanded || isRepliesLoading) {
+            return null;
+        }
+
+        if (repliesTotalCount > repliesLength) {
+            return (
+                <PlainButton
```

Comment:
Using separate buttons for a toggle will cause the focus to transfer to the document, which is a poor user experience. Let's use a single button and swap the contents, instead.